nf_core/modules/lint/main_nf.py

```diff
@@ -257,9 +257,27 @@ def check_process_section(self, lines, fix_version, progress_bar):
         self.warned.append(("process_standard_label", "Process label unspecified", self.main_nf))
     for i, l in enumerate(lines):
         url = None
+        l = l.strip(" '\"")
         if _container_type(l) == "bioconda":
             bioconda_packages = [b for b in l.split() if "bioconda::" in b]
-        l = l.strip(" '\"")
+        if _container_type(l) == "bioconda" or _container_type(l) == "conda":
```

Comment:
The function _container_type() can return "bioconda", "docker" or "singularity", so you can remove this if and use the same one from the previous line

Reply:
`_container_type()` no longer returns `bioconda`, instead it returns `conda`.